plugins/mcreator-core/procedures/get_command_parameters.json

```diff
@@ -15,7 +15,7 @@
     ],
     "dependencies": [
       {
-        "name": "cmdparams",
+        "name": "command",
```

Comment:
I already explained it a few messages ago, but it was mainly to make users know there is a difference between `cmdparams` and `cmdargs`. However, I did not know it could break old procedures.

> > These are not procedure blocks but command blocks, also _cmdargs_ was more descriptive that _arguments_...
> 
> tell me the difference between `cmdargs` and `cmdparams`. If you have some difficulties, think how hard it will be for users to understand the difference. Also, `cmdargs` compared to `arguments` is not more descriptive... This means and describes the same thing, the same way. This is just much clear for users.